### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MathyouMB**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,8 @@ flowchart TB
     2402-->3000
     2402-->3005
     2402-->3007
-    2402-->3804
```
Comment (AngelOnFira) [inline README.md:42]: Can we add this line back?

image

**Review by AngelOnFira — CHANGES_REQUESTED** [review]
Just one fix, then should be good :)

**Comment by AngelOnFira** [comment]
Thanks for verifying the rest thought 💯

File: README.md (hunk 1 above)
Comment (MathyouMB) [reply README.md:42]: oof sorry I thought that said 3004 lol

**Review by AngelOnFira — APPROVED** [review]

**Pull request merged**